R/mrmash_wrapper.R

```diff
@@ -115,15 +117,32 @@ mrmash_wrapper <- function(X,
                            tol = 0.01, 
                            verbose = FALSE){
 
-  if (is.null(prior_grid)) {
-    stop("Please provide prior grid.")
-  }
 
   if (is.null(prior_data_driven_matrices) && !isTRUE(prior_canonical_matrices)) {
     stop("Please provide prior_data_driven_matrices or set prior_canonical_matrices=TRUE.")
   }  
+  
+    
+  Y_has_missing <- any(is.na(Y))
+    
+  if(Y_has_missing && B_init_method=="glasso"){
+        stop("B_init_method=glasso can only be used without missing values in Y")
```

Comment:
Should we provide some more hints and tips to users to guide them what to do in this situation?